src/gpuhunt/providers/cudocompute.py

```diff
@@ -0,0 +1,458 @@
+import asyncio
+from collections import namedtuple
+from concurrent.futures import ThreadPoolExecutor
+from itertools import chain
+from typing import Optional, List
+from cudo_compute import cudo_api
+from gpuhunt import QueryFilter, RawCatalogItem
+from gpuhunt._internal.constraints import KNOWN_GPUS
+from gpuhunt.providers import AbstractProvider
+import logging
+
+CpuMemoryGpu = namedtuple("CpuMemoryGpu", ["cpu", "memory", "gpu"])
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+class CudoComputeProvider(AbstractProvider):
+    NAME = "cudocompute"
```

Comment:
It would be great to rename all occurrences to just `cudo`. Both filenames and names in the code

Reply:
Done